test/ode/rkc_tests.jl

```diff
@@ -29,3 +29,26 @@ end
     @test sim.𝒪est[:l∞] ≈ 2 atol=testTol
   end
 end
+
+@testset "Power Iteration of Runge-Kutta-Chebyshev Tests" begin
```

Comment:
You don't need to make a new test set here. Just append it in the above tests.

Reply:
> You don't need to make a new test set here. Just append it in the above tests.

Updated.